nubis/puppet/consul.pp

```diff
@@ -69,6 +69,13 @@
       verbose => true,
       redownload => true, # The file already exists, we replace it
```

Comment:
$mode => '0755',

Looks like wget::fetch supports a mode argument, so wouldn't that be simpler?

Good catch on the file permissions!